src/View/Helper/UrlHelper.php

```diff
@@ -40,6 +40,21 @@ public function build($url = null, $full = false)
         return h(Router::url($url, $full));
     }
 
+    public function imageUrl($path, array $options = [])
+    {
+        return $this->assetUrl($path, $options + ['pathPrefix' => Configure::read('App.imageBaseUrl')]);
+    }
+
+    public function cssUrl($path, array $options = [])
+    {
+        return $this->assetUrl($path, $options + ['pathPrefix' => Configure::read('App.cssBaseUrl'), 'ext' => '.css']);
```

Comment:
This and other similar statement are a bit too long. Would be nice to split them across multiple lines.